### DISCUSSION TITLE
How can I get the internal_messages sub-message data after v16

### DISCUSSION THREAD

**Discussion by LINJINTIANDE** [filecoin-project/lily#1020] [General] [answered]
Before v16, I used the internal_messages sub-message to get the data of the reward amount, but after v16, all the data of the upgrade fvm sub-message are empty, how can I get the data of the sub-message 🙏
![image](https://user-images.githubusercontent.com/42436479/177589774-df9446fb-a143-48fd-afdb-3aed07c19ad4.png)

**Comment by frrist** [comment, marked as answer]
You'll need to set `LOTUS_VM_ENABLE_TRACING=1` when starting the daemon. Will add this to documents and the lily command shortly.
